features/step_definitions.rb

```diff
@@ -14,24 +14,24 @@
 
 #
 
-Given %r{^I have a blank site in "(.*)"$} do |path|
-  if !File.exist?(path)
+Given(/^I have a blank site in "(.*)"$/) do |path|
```

Comment:
Hm, actually it failed with %r!!, I guess because we have following in .rubocop.yml
```yaml
Style/RegexpLiteral:
  EnforcedStyle: slashes
```
Should I change it to `percent_r` ?